judo-runtime-core-validator/src/main/java/hu/blackbelt/judo/runtime/core/validator/RangeValidator.java

```diff
@@ -70,7 +70,8 @@ public Collection<ValidationResult> validateValue(Payload instance, final EStruc
         @SuppressWarnings("unchecked")
         final Collection<Payload> range = dao.getRangeOf((EReference) feature, instance, DAO.QueryCustomizer.<ID>builder()
                 .withoutFeatures(true)
-                .build());
+                .build(),
+                false);
```

Comment:
Should be true

Reply:
In this case, there is no rollback, so we cannot create a new instance or update an existing one.